config/application.rb

```diff
@@ -43,5 +43,10 @@ class Application < Rails::Application
     config.autoload_paths += Dir[Rails.root.join('app', '**/')]
 
     config.active_record.raise_in_transactional_callbacks = true
+
+    Yt.configure do |config|
```

Comment:
I think this section of code should be in a separate file called yt.rb or similar